src/Compute/Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeCancelMethod.cs

```diff
@@ -48,7 +48,16 @@ public override void ExecuteCmdlet()
                     string resourceGroupName = this.ResourceGroupName;
                     string vmScaleSetName = this.VMScaleSetName;
 
-                    var result = VirtualMachineScaleSetRollingUpgradesClient.CancelWithHttpMessagesAsync(resourceGroupName, vmScaleSetName).GetAwaiter().GetResult();
+                    Rest.Azure.AzureOperationResponse result;
+                    if (NoWait.IsPresent)
+                    {
+                        result = VirtualMachineScaleSetRollingUpgradesClient.BeginCancelWithHttpMessagesAsync(resourceGroupName, vmScaleSetName).GetAwaiter().GetResult();
+                    }
+                    else
+                    {
+                        result = VirtualMachineScaleSetRollingUpgradesClient.CancelWithHttpMessagesAsync(resourceGroupName, vmScaleSetName).GetAwaiter().GetResult();
+                    }
+
```

Comment:
Same - I am not marking any more of these - lets find all uses of GetOperationIdFromUrlString and make sure we are not adding NoWait to any of them.